### PR TITLE
Setup hello world project

### PR DESCRIPTION
**Pull request by CindyDorantes**
**First request for code review.** Changes introduced:
- create HTML file
- create CSS file
- create test.md

**Second request for code review.** Changes introduced:
- Modify README file according to template
- Modify .gitignore with correct file names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,97 @@
...
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
```
Comment (zainsadaqat) [inline README.md]: Please use the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @CindyDorantes  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CindyDorantes) [reply README.md]: Hi! I have updated the README file. @zainsadaqat could you please review it?
Thanks!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
-# hello_world
-Project Hello World week 1
+Project Name
```
Comment (Ismailco) [inline README.md]: - [ ] The README looks better but as the previous reviewer mentioned, please consider using the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse, and don't forget to customize it.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+test
```
Comment (Ismailco) [inline .gitignore]: - [ ] [OPTIONAL] The file name is ```test.md``` please consider using the proper naming so git will not push the file to Github, this is required

File: README.md (hunk 2 above)
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] This Pull Request should have a descriptive title telling you what changes are inside e.g: ``` Setup hello world project``` and the description (The first comment in the PR) should have a summary describing introduced changes in more detail, e.g: 
    ```
    - create HTML file
    - create CSS file
    - create test.md
    ...
    ```

Comment (Ismailco) [inline README.md]: - [ ] [Optional] The branch should have the name of a feature or action e.g. ```'created_sidebar'``` or ```'added_comments_section'```, please consider doing this in future projects, you can skip this for now

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @CindyDorantes,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (CindyDorantes) [reply README.md]: @Ismailco Does that mean that I shouldn't name the branch as "feature", but instead I should use a more descriptive name?

Comment (CindyDorantes) [reply README.md]: @Ismailco Indeed I used the readme template, but I deleted some sections. So I understand I shouldn't have deleted any section, but instead just customize the existing sections, rigth?

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @CindyDorantes 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**